src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img class="product-card__img"
+        src="images/imac.jpeg"
+        alt=""
+      >
+
+      <div class="product-card__description">
+        <h3 class="description-title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+
+        <div class="description-code">Код товара: 195434</div>
+
+        <div class="reviews">
+          <div class="stars">
```

Comment:
```suggestion
          <div class="stars stars--4">
```
Тут еще должен быть класс-модификатор, который отвечает за количество активных звездочек